src/http.cpp

```diff
@@ -398,8 +404,11 @@ int Exiv2::http(Exiv2::Dictionary& request, Exiv2::Dictionary& response, std::st
             //  we finished OK without finding headers, flush the buffer
             flushBuffer(buffer, 0, end, file);
         } else {
+            auto errorCode = WSAGetLastError();
```

Comment:
I thought you added that label intentionally to use the new mergrify service. @D4N merged recently  #673 and if you add that label, once you merge this into **master** it will try to merge it to 0.27 too.

Actually, I do not know if it creates a PR automatically or what ... @D4N do you know what would happen in that case?